streamer/src/quic.rs

```diff
@@ -400,14 +401,14 @@ pub fn spawn_server(
     staked_nodes: Arc<RwLock<StakedNodes>>,
     max_staked_connections: usize,
     max_unstaked_connections: usize,
-    stats: Arc<StreamStats>,
     wait_for_chunk_timeout: Duration,
     coalesce: Duration,
 ) -> Result<(Endpoint, thread::JoinHandle<()>), QuicServerError> {
     let runtime = rt();
-    let (endpoint, task) = {
+    let (endpoint, _stats, task) = {
         let _guard = runtime.enter();
         crate::nonblocking::quic::spawn_server(
+            name,
```

Comment:
I think we can also use the name in line 426 to make the quic servers threads have unique name for debugging purpose. With that -- I think we may need shorter names?

Reply:
That is a good point but unix thread names are limited to 15 characters.
We are also using different convention for thread names vs metrics names.
So leaving this unchanged for now.